randovania/games/samus_returns/json_data/Area 3 - Exterior.txt

```diff
@@ -731,35 +731,13 @@ Extra - asset_id: collision_camera_038
   * Extra - actor_name: Door015
   * Extra - actor_type: doorpowerpower
   > Event - Gamma Metroid
-      All of the following:
-          Defeat Gamma Metroid and Lay Any Bomb
-          Walljump (Beginner) or Climb Rooms Vertically (High Jump)
+      Climb Rooms Vertically (High Jump) and Defeat Gamma Metroid and Lay Any Bomb
```

Comment:
Why remove beginner WJ? Is the metroid on the top now, not the right?

Reply:
Yes these metroid changes follow the map update and patcher changes.